dotcom-rendering/index.d.ts

```diff
@@ -1065,9 +1065,9 @@ interface PerformanceEntry {
 
 declare namespace JSX {
 	interface IntrinsicElements {
-		'gu-hydrate': {
+		'gu-island': {
 			name: string;
-			when?: 'immediate' | 'idle' | 'visible';
+			defer?: 'idle' | 'visible';
```

Comment:
How about `deferUntil`? It seems to read better.

Reply:
Done